test/test_factory_helper_name.rb

```diff
@@ -3,18 +3,38 @@
 class TestFactoryHelperName < Test::Unit::TestCase
 
   def setup
+    FactoryHelper::Config.locale = nil
     @tester = FactoryHelper::Name
   end
-  
+
+  def teardown
+    FactoryHelper::Config.locale = nil
+  end
+
   def test_name
     assert @tester.name.match(/(\w+\.? ?){2,3}/)
   end
-  
+
   def test_prefix
     assert @tester.prefix.match(/[A-Z][a-z]+\.?/)
   end
-  
+
   def test_suffix
     assert @tester.suffix.match(/[A-Z][a-z]*\.?/)
   end
+
+  def test_girls_name_en_US
+    FactoryHelper::Config.locale= 'en-US'
+    1200.times do
+      assert @tester.girls_name.match(/^([A-Z])(\w)*$/)
+    end
+  end
+
+  def test_boys_name_en_US
+    FactoryHelper::Config.locale= 'en-US'
+    300.times do
+      assert @tester.boys_name.match(/^([A-Z])(\w)*$/)
+    end
+  end
```

Comment:
So, based on our discussion, what about separating test of values validity from test of ability to access said values:
```ruby
en_us_locale = Psych.load(File.read('lib/locales/en-US.yml'))
boys_names = en_us_locale['en-US']['factory_helper']['name']['boys_name']
assert_equal true, boys_names.map { |n| n.match(/\A[A-Z][\w']*/) }.all? # apostrophe to match "D'Angelo" in `en` locale, might need to adjust further for the `en-US` names

assert @tester.boys_name.match(/^([A-Z])(\w)*$/) # 1 time only to test retrieval
```